plugins/usbdmx/EurolitePro.cpp

```diff
@@ -176,6 +178,26 @@ bool SynchronousEurolitePro::Init() {
     return false;
   }
 
+  // USB-DMX512-PRO MK2: set baudrate to 250000
+  if (m_is_mk2) {
+    uint16_t divisor = 12;  // = 3000000 / 250000
+    uint16_t value = (divisor & 0xFFFF);
```

Comment:
This is achieving nothing, as divisor is already a uint16, did you mean to bitwise and with 0x00FF?

Reply:
I didn't write that part, but I think that normally the divisor should indeed be AND by 0xFFFF (e.g. I checked the [libftdi code](http://developer.intra2net.com/git/?p=libftdi;a=blob_plain;f=src/ftdi.c;hb=HEAD) and there the "value" parameter is also & by `0xFFFF`. That's necessary when supporting any divisor, but since our divisor is already known and already an `uint16_t`, it indeed is not necessary. I'll remove the & and add a comment. This could of course be further reduced because value and index can be set directly to their end value, but having the `divisor` variable is useful for documentation and I assume the compiler can optimize it out.

Reply:
0xFFFF now removed and comment added.